pimple.js

```diff
@@ -125,7 +129,7 @@
         this.define('pimple',[],function(){return self.Pimple;});
     }
     //CommonJS
-    if(module && module.exports){
+    if(typeof exports !== 'undefined' && module && module.exports){
```

Comment:
Why do you need ``exports`` do be defined? Isn't is enough with ``module.exports``?